src/test/java/myPackage/Driver.java

```diff
@@ -5,7 +5,7 @@
 
 public class Driver extends Elements{
 
-    public ChromeDriver driver() {
+    public WebDriver driver() {
         // opens chrome browser
         System.setProperty("webdriver.chrome.driver", "drivers/chromedriver");
         ChromeDriver browser = new ChromeDriver();
```

Comment:
WebDriver browser = new ChromeDriver();

Reply:
Ok, I will use ChromeDriver browser = new ChromeDriver(); instead of WebDriver.